docs/dev/first-contribution.rst

```diff
@@ -0,0 +1,213 @@
+The Guide to your First Contribution to Pontoon
+===============================================
+
+Welcome to Pontoon!
+
+This document is going to guide you as you discover Pontoon and make
+valuable contributions. It will walk you step by step until you are in a
+position to write code that you can reliably run, test, and send for
+review.
+
+1. Check your skills
+--------------------
+
```

Comment:
This feels a bit 'hard' , maybe just "Skills check"  or "Check if you have the needed skills"